CHANGELOG.md

```diff
@@ -0,0 +1,18 @@
+# Change Log
+
+All notable changes to this project will be documented in this file.
+
+## Unreleased - as of Sprint 52 end 2017-01-14
+
+### Added
+- Inventory collection refresh using inventory abstraction ([#102](https://github.com/ManageIQ/manageiq-providers-azure/pull/Inventory collection refresh using inventory abstraction))
```

Comment:
Wow that's a confusing PR title.  How about:

"Collect inventory using 'inventory' abstraction in refresh"